package.json

```diff
@@ -26,21 +26,21 @@
     "@actions/core": "^1.4.0",
     "@actions/exec": "^1.1.0",
     "@actions/github": "^5.0.0",
-    "@types/jest": "^26.0.23",
-    "@zeit/ncc": "^0.22.3",
     "fs": "0.0.1-security",
-    "jest": "^26.6.3",
-    "jest-junit": "^12.2.0",
-    "request": "^2.88.2",
-    "ts-jest": "^26.5.6",
-    "typescript": "^4.3.2",
-    "yarn": "^1.22.10"
+    "request": "^2.88.2"
   },
   "devDependencies": {
+    "@types/jest": "^26.0.23",
     "@types/node": "^14.17.3",
     "@typescript-eslint/eslint-plugin": "^4.26.1",
     "@typescript-eslint/parser": "^4.26.1",
+    "@zeit/ncc": "^0.22.3",
     "eslint": "^7.28.0",
-    "eslint-config-google": "^0.14.0"
+    "eslint-config-google": "^0.14.0",
```

Comment:
This is unused but it doesn’t have reason to be. Can the current configuration make use of it?

Reply:
I believe this is being used [here](https://github.com/codecov/codecov-action/blob/master/.eslintrc.json#L8)